Source/WebCore/platform/graphics/avfoundation/CDMPrivateMediaSourceAVFObjC.h

```diff
@@ -52,7 +52,7 @@ class CDMPrivateMediaSourceAVFObjC final : public CDMPrivateInterface, public Ca
     bool supportsMIMEType(const String& mimeType) const override;
     RefPtr<LegacyCDMSession> createSession(LegacyCDMSessionClient&) override;
 
-    LegacyCDM& cdm() const { return m_cdm.get(); }
+    LegacyCDM& cdm() const;
```

Comment:
Hopefully this doesn't introduce a whole lot of safer cpp failures...

Reply:
The safer cpp bot seems ok in last run (this change is needed to avoid including header, or let me know if you prefer including header).